ansible/templates/supervisor.gunicorn.conf.j2

```diff
@@ -1,19 +1,15 @@
 # {{ ansible_managed }}
 # Last run: {{ template_run_date }}
-
 [supervisord]
 environment=LC_ALL='en_US.UTF-8',LANG='en_US.UTF-8'
-
 [program:{{ supervisor_gunicorn_app }}]
 directory=/var/www/{{ tgwf_domain_name }}.thegreenwebfoundation.org/current/
 numprocs=1
 command=bash ./run_gunicorn.sh
 process_name=%(process_num)02d
-environment=PORT=90%(process_num)02d
```

Comment:
We don't scale with supervisor - gunicorn has good support for forking subprocesses and threads, so we do it there instead.